src/www/ui/api/Controllers/AuthController.php

```diff
@@ -56,6 +56,15 @@ public function getAuthHeaders($request, $response, $args)
       $returnVal->getArray(), $returnVal->getCode());
   }
 
+  public function optionsVerification($request, $response, $args)
+  {
+    $Message = "Successful";
+    $returnVal = new Info(200, $Message, InfoType::INFO);
+    return $response->withJson(
+      $returnVal->getArray(), $returnVal->getCode());
```

Comment:
In my opinion, a better response will be code `204`, no body and with headers `Access-Control-Allow-Origin` and `Access-Control-Allow-Methods` set.
Adding headers here will eliminate need to add them in Apache's configuration.

Reply:
Also, it will be nice to add the `Access-Control-Allow-Origin` from Admin > Customize instead of hard coding `*`.
For reference, see #1890 for changes in `src/lib/php/common-sysconfig.php` and `src/www/ui/page/UploadPageBase.php`

Reply:
The headers mentioned will be a good addition. Some points:
1. The value for `Access-Control-Allow-Origin` should come from `sysconfig` table (not from fossology.conf), not hard-coded.
2. For the values of `Access-Control-Allow-Headers` mentioned here, there is a `*` at the end. Won't that make it equivalent to `Access-Control-Allow-Headers  "*"`?
3. FOSSology also uses `PUT, PATCH` methods (please check [the openapi.yaml](https://github.com/fossology/fossology/blob/master/src/www/ui/api/documentation/openapi.yaml)).